src/commands/sourcemaps/git.ts

```diff
@@ -182,6 +182,10 @@ export class TrackedFilesMatcher {
   // Example: webpack:///./src/folder/ui/select.vue?821e
   private getFilename(s: string): string {
     let start = s.lastIndexOf('/')
+    const backslashStart = s.lastIndexOf('\\')
```

Comment:
What if we have a unix file with a backslash in its name?

Reply:
Those particular files won't be reported.  Not sure there are better ways to handle that case.
We could use the current env separator but the paths in the sourcemaps 'sources' could still use any separator.